packages/serialize/types/tests.ts

```diff
@@ -16,36 +16,33 @@ const testObjectInterpolation1: ObjectInterpolation<undefined> = {
 }
 
 // $ExpectType SerializedStyles
-serializeStyles({}, [])
+serializeStyles([], {})
```

Comment:
why the arguments order has changed here and at other places in this file? 😄

Reply:
They got reordered in #1236, but the tests were not updated.